index.html

```diff
@@ -6,8 +6,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Home</title>
 
-    <link rel="stylesheet" href="./main.css">
-    <link rel="stylesheet" href="./Global-styles.css">
+    <link rel="stylesheet" href="./CSS/Global-styles.css">
```

Comment:
I remember Johan told us that typography go first, but the code is perfect. Nice job Jefry Mauricio.

Reply:
I'll make the changes